src/styles/styles.js

```diff
@@ -743,11 +741,20 @@ const styles = {
     },
 
     hamburgerOpen: {
-        borderRightWidth: 1,
         borderColor: themeColors.border,
         width: 300,
     },
 
+    w300br1: {
+        width: 300,
+        borderRightWidth: 1,
+    },
+
+    w0br0: {
+        width: 0,
+        borderRightWidth: 0,
+    },
```

Comment:
I think for now we can just update these style names to `sidebarHidden` and `sidebarVisible`. 
After this change is applied I will merge the solution.

Reply:
@marcaaron 
Updated